skit_labels/constants.py

```diff
@@ -120,4 +120,10 @@
 FROM_NAME_INTENT = "tag"
 CHOICES = "choices"
 TAXONOMY = "taxonomy"
-VALUE = "value"
\ No newline at end of file
+VALUE = "value"
+
+EXPECTED_COLUMNS_MAPPING = {
+    "conversation_tagging": ['scenario', 'scenario_category', 'situation_str', 'call', 'data_label']
```

Comment:
This works for now but ideally even field type will be checked in the future. So something like:
"conversation_tagging": {'scenario', 'scenario_category':<int>, 'situation_str':<str>, 'call':<str>, 'data_label':<str>}

But this is good for now

Reply:
Will add this in the next iteration, when I add the changes for the download functions.